.evergreen/run-atlas-search-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -o errexit
+
+# Supported/used environment variables:
+# MONGODB_URI Set the connection to an Atlas cluster
+
+############################################
+#            Main Program                  #
+############################################
+RELATIVE_DIR_PATH="$(dirname "${BASH_SOURCE[0]:-$0}")"
+source "${RELATIVE_DIR_PATH}/javaConfig.bash"
+
+echo "Running Atlas Search tests"
+./gradlew -version
+#./gradlew --stacktrace --info \
+#    -Dorg.mongodb.test.atlas.search=true \
+#    -Dorg.mongodb.test.uri=${MONGODB_URI} \
+#    driver-core:test --tests AggregatesSearchIntegrationTest
```

Comment:
Commented out until [CLOUDP-116195 Atlas cluster for drivers to test search APIs](https://jira.mongodb.org/browse/CLOUDP-116195) is done.

One may run Atlas search integration tests locally by uncommenting this code and replacing `${MONGODB_URI}` with `mongodb+srv://driver-testing:JIbS2Flc30ENAOL0@free-cluster-search.ujz9d.mongodb.net`.

Reply:
Configured a cluster for `AggregatesSearchIntegrationTest` here https://cloud-dev.mongodb.com/v2/5dd7496c7a3e5a648454341b#clusters and uncommented the Bash script. Let's see if the tests pass on Evergreen (they passed when I started them locally).

Reply:
And [it worked](https://spruce.mongodb.com/task/mongo_java_driver_atlas_search_test_atlas_search_test_patch_3928344ab5738f62d563bbb6bc86191e82b1fa80_624b36097742ae73b4aae2c7_22_04_04_18_16_43/tests?execution=0&sortBy=STATUS&sortDir=ASC). Resolving.